packages/sitecore-jss/src/graphql/app-root-query.ts

```diff
@@ -65,12 +65,20 @@ export async function getAppRootId(
 
   debug.dictionary('fetching site root for %s %s', language, siteName);
```

Comment:
Noticed this debug log for dictionary namespace here. We should move this to the `GraphQLDictionaryService` (just before the `getAppRootId` call) instead.